modules/instrument_list/templates/menu_instrument_list.tpl

```diff
@@ -15,10 +15,21 @@
     <th>
       Biological Sex
     </th>
-    {if $display.ProjectTitle != ""}
+    {if $display.ProjectTitle == $display.ProjectName && $display.ProjectName != ""}
       <th>
         Project
       </th>
+    {else}
+      {if $display.ProjectTitle != ""}
+        <th>
+          Candidate Project
```

Comment:
The wording of this is a little confusing. Do the timepoints not belong to the candidate? Can we label it "Candidate Registration Project" or if that takes up too much space just "Registration Project"?